File: Problem 7/sol.js
function diagonalDifference(arr) {
    const size = arr.length;
    let sumLeft = 0;
    let sumRight= 0;
    
    for(let i=0;i<size;i++){
        sumLeft += arr[i][i];
        sumRight+= arr[size-1-i][i];
    }
    
    return Math.abs(sumLeft-sumRight);

}

File: Problem 7/sol.cpp
int diagonalDifference(vector<vector<int>> arr) {
    int co = arr.size();
    int sumaLeft = 0;
    int sumaRight = 0;
    
    for(int i=0;i<co;i++){
        sumaLeft+=arr[i][i];    
        sumaRight+=arr[co-i-1][i];
    }
    
    return abs(sumaRight - sumaLeft);
}